python_modules/libraries/dagster-fivetran/dagster_fivetran_tests/experimental/conftest.py

```diff
@@ -61,7 +61,7 @@
                 },
                 "config": {"property1": {}, "property2": {}},
                 "daily_sync_time": "14:00",
-                "succeeded_at": "2024-12-01T15:43:29.013729Z",
+                "succeeded_at": "2024-12-01T15:45:29.013729Z",
```

Comment:
why the slight timing changes?

Reply:
I updated the `failed_at` as well so that it is before the `succeeded_at`. 

In the poll process, we confirm that the sync succeed if:

- the value of `succeeded_at` of the current API response is greater than the one we had at the beginning of the process
- the `succeeded_at` is received before the poll process times out
- the value of `succeeded_at` is greater that the value of `failed_at`

The example provided by Fivetran had a `failed_at` value greater than the `succeeded_at`, so I inverted them here.

Reply:
Gotcha, that makes sense.